tools/cli/src/opt/kanidm.rs

```diff
@@ -602,7 +603,7 @@ pub struct Oauth2CreateScopeMapOpt {
     nopt: Named,
     #[clap(name = "group")]
     group: String,
-    #[clap(name = "scopes")]
+    #[clap(name = "scopes")] // TODO: is this required, or is null implying a default?
```

Comment:
Null implies delete here IIRC.

Reply:
cool, leaving it unchanged for now then.